### PR TITLE
Refactor based on new website.

### PR DESCRIPTION
**Pull request by daniel-veecle**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,28 +1,47 @@
...
+We integrate to the [Robot Operating System](ros) to allow for easy integration with enabled devices over the [Data Distribution Service](dds). 
 
-## Learn more
-Are you interested in Veecle? \
-Visit us at www.veecle.io or write us at info@veecle.io
+This is not public (yet). You can find information [here](ros-plugin).
+
+## About Veecle
+
+Veecle sits between a legacy E/E architecture and modern UI and Cloud frameworks. Veecle abstracts an E/E architecture and provides unified control over it using a modern API. Veecle allows companies to develop "Apps" using unified APIs to interact with in-vehicle functions. An optional front-end framework is provided through Android Automotive support.
```
Comment (alexveecle) [inline profile/README.md:28]: I do not know what E/E means, but perhaps I'm just not familiar with the industry. Perhaps it should be expanded.

As I have already opened a comment:

```suggestion
Veecle sits between a legacy E/E architecture, and modern UI and Cloud frameworks. Veecle abstracts an E/E architecture and provides unified control over it using a modern API. Veecle allows companies to develop applications using unified APIs to interact with in-vehicle functions. An optional front-end framework is provided through Android Automotive support.
```

(I don't think capitalization nor quotes of "apps" is needed. I prefer "application", but that's a personal taste.)

File: profile/README.md [hunk 2]
```diff
@@ -1,28 +1,47 @@
...
-- **Veecle Studio & Cloud Services** consist of a set of tools that can be used to deploy, test, monitor and manage our instances. At its core, this is a set of secure APIs that allows interconnecting visual tools for development, deployment, data visualisation, data storage and fleet management
+- [Veecle-pxros](https://github.com/veecle/veecle-pxros) A runtime for writing Rust applications on top of HighTec's safety kernel for the Infineon AURIX™ Platform.
+- [Tricore-probe](https://github.com/veecle/tricore-probe): A tool that facilitates the flashing and debugging of Rust code on AURIX™ chips.
 
-## What is currently published
+### Android Automotive support
 
-- [TC37x-demo](https://github.com/veecle/tc37x-demo): demo code to run a 100% Rust basic CAN driver on AURIX™ TC375 LK
-- [Tricore-probe](https://github.com/veecle/tricore-probe): a Rust probe for AURIX™ (currently works on Windows and Linux)  
+We support [Android Automotive](aaos): this enables customers to develop safe and efficient interfaces to the vehicle.
```
Comment (alexveecle) [inline profile/README.md:16]: The link label is not being resolved if I view the file here:

https://github.com/veecle/.github/blob/b7831ef6e59043330de3d79e6e1b20a70b70b3ef/profile/README.md

I think this happens with the rest of links.

**Review by alexveecle — APPROVED** [review]

File: profile/README.md [hunk 3]
```diff
@@ -1,28 +1,40 @@
 # Veecle
 
-Veecle | All Cars. One Software. Our mission is to bridge the gap between existing supplier components and smart apps running on top through unified software. We provide a platform to tackle next-gen automotive connected & security demands. 
+Our mission is to bridge the gap between existing supplier components and smart apps running on top through unified software framework. We provide a platform to tackle next-gen automotive connected & security demands.
```
Comment (arctic-alpaca) [inline profile/README.md:3]:
```suggestion
Our mission is to bridge the gap between existing supplier components and smart apps running on top through a unified software framework. We provide a platform to tackle next-gen automotive connected & security demands.
```

File: profile/README.md [hunk 4]
```diff
@@ -1,28 +1,40 @@
...
+We integrate to the [Robot Operating System](https://www.ros.org) to allow for easy integration with enabled devices over the [Data Distribution Service](https://www.dds-foundation.org/what-is-dds-3). 
 
-## Learn more
-Are you interested in Veecle? \
-Visit us at www.veecle.io or write us at info@veecle.io
+This is not public (yet). You can find information [here](https://veecle.io/tech/dds).
+
+## About Veecle
+
+Veecle sits between a legacy electrical/electronic (E/E) architecture, and modern UI and Cloud frameworks. Veecle abstracts an E/E architecture and provides unified control over it using a modern API. Veecle allows companies to develop applications using unified APIs to interact with in-vehicle functions. An optional front-end framework is provided through Android Automotive support.
```
Comment (arctic-alpaca) [inline profile/README.md:28]:
```suggestion
Veecle
* sits between a legacy electrical/electronic (E/E) architecture, and modern UI and Cloud frameworks.
* abstracts an E/E architecture and provides unified control over it using a modern API.
* allows companies to develop applications using unified APIs to interact with in-vehicle functions. An optional front-end framework is provided through Android Automotive support.
```

Its a lot of Veecle does.

File: profile/README.md [hunk 5]
```diff
@@ -1,28 +1,40 @@
...
+
+Our runtime seamlessly integrates across many ECUs — from infotainment systems to real-time controllers, playing together with existing and external services. The very same Rust code base can be compiled for Linux (e.g. Android Automotive) or as an isolated task on a real-time operating system (RTOS), ensuring *freedom from interference* on e.g. Infineon AURIX™ chips, or ARM embedded controllers.
```
Comment (arctic-alpaca) [inline profile/README.md:30]:
```suggestion
Our runtime seamlessly integrates across many ECUs — from infotainment systems to real-time controllers, cooperating with existing and external services. The very same Rust code base can be compiled for Linux (e.g. Android Automotive) or as an isolated task on a real-time operating system (RTOS), ensuring *freedom from interference* on e.g. Infineon AURIX™ chips, or ARM embedded controllers.
```

**Review by arctic-alpaca — COMMENTED** [review]
Some nitpicks but overall good :+1:

**Review by HectorMRC — APPROVED** [review]
LGTM

File: profile/README.md (hunk 4 above)
Comment (arctic-alpaca) [reply profile/README.md:28]: It's fine but not really a nice paragraph structure. If possible, this should be changed on the website too.

**Review by arctic-alpaca — APPROVED** [review]

**Comment by Elexir-Stefan** [comment]
Looks fine. Thanks 👍👍

**Pull request merged**